database_init.sql

```diff
@@ -152,7 +152,7 @@ CREATE TABLE STUDENT_TYPE(
 ) ENGINE=InnoDB;
 
 CREATE TABLE APPLICATION(
-  APP_ID INT NOT NULL PRIMARY KEY,
```

Comment:
this is the only change in this PR that hasn't been made already, should probably be added if it's required.